### ISSUE TITLE
RuntimeError: 'Missing "jwks_uri" in metadata'

### ISSUE THREAD

**Issue by InnovativeInventor** [authlib/loginpass#59]
I tried using the exact code from `loginpass/flask_example` and got the following errors:

```bash
[2020-02-21 18:32:38,624] ERROR in app: Exception on /google/auth [GET]
Traceback (most recent call last):
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/loginpass/_flask.py", line 60, in auth
    user_info = remote.parse_openid(token, nonce)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/loginpass/google.py", line 50, in parse_openid
    token.get('access_token'), nonce
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/loginpass/_core.py", line 115, in parse_id_token
    claims_params=claims_params,
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/jose/rfc7519/jwt.py", line 121, in decode
    data = self._jws.deserialize_compact(s, key_func, decode_payload)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/jose/rfc7515/jws.py", line 115, in deserialize_compact
    self._algorithms, jws_header, payload, key)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/jose/util.py", line 8, in prepare_algorithm_key
    key = key(header, payload)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/jose/jwk.py", line 27, in key_func
    return load_key(key, header, payload)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/jose/jwk.py", line 21, in load_key
    key = key(header, payload)
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/loginpass/_core.py", line 95, in load_key
    jwk_set = remote.fetch_jwk_set()
  File "/Users/max/.pyenv/versions/anaconda3-2019.10/lib/python3.7/site-packages/authlib/integrations/base_client/remote_app.py", line 137, in fetch_jwk_set
    raise RuntimeError('Missing "jwks_uri" in metadata')
RuntimeError: Missing "jwks_uri" in metadata
```
Is this a problem with the library? My creds were correct, I believe (and I set the google oauth API to allow redirects to localhost, etc.).

**Comment by asheinfeld** [comment]
I'm having the same issue mentioned by @InnovativeInventor

**Comment by erickhun** [comment]
The latest changes in  `Authlib` (0.13+) are unfortunately  incompatible with the current release of loginpass. 

The quick fix for now to use Authlib 0.12.1 , it's as simple as : 

```bash
pip install Authlib==0.12.1
```

More details on the why this happen : https://github.com/authlib/loginpass/issues/56 

cc @asheinfeld  @InnovativeInventor

**Comment by lepture** [comment]
v0.5 released
